02-household-queries/qt_summaries-gpt-3.5-turbo-instruct-k_4.json

```diff
@@ -0,0 +1,851 @@
+                    "Do you provide regular support for the grandchild?",
+                    "Is the grandchild financially dependent on you?"
+                ],
+                "entire_card": "In what cases can a parent who has a joint custody arrangement include a child on a food stamp application?\nAnswer:\n(a) When the custody arrangement between parents is uneven, the child would be included in the household that provides the child with the majority of his or her meals.\n(b) When there is an equal custody arrangement and the children eat 50 percent of their meals with each parent, the case manager must decide which household should include the child. If both parents apply and cannot agree on which household will include the child, the case manager will resolve the issue by including the child in the household that applied first.\nNote: The case manager must insure that the child is included in only one household.\nFood Stamps only",
+                "summary": "The grandchild should be included in the household that provides the majority of their meals, or if there is an equal custody arrangement, the household that applied first."
+            },
+            "Can students be part of a SNAP household?": {
+                "score_sum": 0.2095682249578389,
+                "derived_questions": [
+                    "Do you provide regular support for the grandchild?"
+                ],
+                "entire_card": "Can students be part of a SNAP household?\nYes, however students must meet additional criteria to be part of the SNAP Household.\nStudents must meet one of the following conditions:\nunder age 18 or age 50 or older\nHave a physical or mental disability\nstill in high school\nresponsible for the care of a dependent household member under age 6\nresponsible for dependent household member aged 6-11 without adequate child care and therefore cannot participate in job training or work requirements\nEnrolled full-time and is a single parent with responsibility for the care of a dependent child under age 12, regardless of the availability of adequate child care\nworking at least 20 hours per week\nreceiving Temporary Cash Assistance benefits\nare attending less than half-time\nparticipating in on the job training",
+                "summary": "Yes, students can be part of a SNAP household if they meet certain criteria, such as being under 18 or over 50, having a disability, or being responsible for a dependent household member."
+            }
+        }
+    },
+    {
+        "id": 2,
+        "question": "The client's son is 20 and is still living with them, but has his own job and buys his own food. The client doesn't want to list him on the application, do they have to? If so, do we need to include the son's income?",
+        "guru_cards": [
+            "Who are mandatory HH members for food stamps?",
+            "How is the income of a dependent counted for SNAP?"
```

Comment:
Since the son is not a minor (and not a dependent), how do we convince the software to retrieve the Guru card that ask about a dependent? We were able to do it by _manually_ rephrasing the question [here](https://github.com/navapbc/labs-gen-ai-experiments/blob/6ddcfe85cb0e9daaefddb50cfc50c68e942d72b8/02-household-queries/question_answer_citations.json#L15) to `... do we need to include the dependent's income for SNAP?`.

Reply:
Furthermore, the Guru card could be better written to avoid this misinterpretation: 
The second card's `How is the income of a dependent counted for SNAP? Counts as income, unless ...` is (incorrectly) interpreted as: "Count the dependents income unless ..."